bow/1e0deee4-7dc1-400f-acb6-74c0f4aec471.md

```diff
@@ -0,0 +1,32 @@
+# bow
+
+Bags of features, extracted in july 2018 from 7.8 million distinct files from PGA (taking only the `HEAD` commit), using all implemented extractors in `sourced.ml` at the time (`identifiers`, `literals`, `graphlets`, `children`, `node2vec` and `uast2seq`) and all languages parsable by Babelfish (Go, Java, Python, Bash, JavaScript and Ruby). This was done to try to use [apollo](https://github.com/src-d/apollo) at scale. We hit `scipy.sparse` limits while trying to merge sparse matrices for all bags, so this is only one of three `BOW` model holding bags.
```

Comment:
`BOW` or `bow`?

Reply:
Should be BOW everywhere. I think we lack proper, long names for the models and use their type ids for the headers.